design/authenticators/authn_gcp/authn_gcp_solution_design.md

```diff
@@ -90,8 +90,7 @@ We will support only the following service accounts types:
 * User-managed service accounts - created by the user
 
 
-As mentioned in the [feature doc](https://ca-il-confluence.il.cyber-ark.com/display/rndp/Conjur+GCP+Authenticator#/), 
-before any authentication request is sent to Conjur, the admin will load the authenticator policy:
+Before any authentication request is sent to Conjur, the admin will load the authenticator policy:
```

Comment:
You can link to the[ GH version](https://github.com/cyberark/conjur/issues/1711) of the design